scripts/postinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ ! -L ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness ]; then
+    mkdir -p ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages
+    ln -s ../../../share/leap_testing/tests/TestHarness ${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages/TestHarness
```

Comment:
Instead of `${CMAKE_INSTALL_FULL_LIBDIR}/python3/dist-packages`, would `${Python_SITELIB}` be more appropriate since the system python installation directory doesn't depend on the installation path of leap?

Reply:
I concur, finding `Python_SITELIB` on the host is probably not optimal.

But we could potentially use `/usr/lib/python3/dist-packages`. It would work for Debian Bookworm and Ubuntu 18 through 23.

Reply:
Issue created to investigate this in the future: https://github.com/AntelopeIO/leap/issues/839